### ISSUE TITLE
[Bug]: Should 'is_ascending_series' skip NaNs?

### ISSUE THREAD

**Issue by CodyCBakerPhD** [NeurodataWithoutBorders/nwbinspector#343]
### What happened?

This was discovered during a particular check run, but the question is more general

I have a time vector (in this case a table column) that has a small number of NaN's in it but most of the time has well-behaved strictly ascending floats

Should [our util function for determining if a time vector is ascending](https://github.com/NeurodataWithoutBorders/nwbinspector/blob/801d8ec97ff238b13e8a6c30a8b8f0b419b00a93/src/nwbinspector/utils.py#L90-L95) filter out NaN values?

### Operating System

Windows

### Python Executable

Conda

### Python Version

3.7

### Usage

Command Line Interface

### Were you streaming with ROS3?

No

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/NeurodataWithoutBorders/nwbinspector/blob/dev/.github/CODE_OF_CONDUCT.rst)
- [X] Have you ensured this bug was not already [reported](https://github.com/neurodatawithoutborders/nwbinspector/issues)?
- [X] To the best of your ability, have you ensured this is a bug within the code that checks the NWBFile, rather than a bug in the NWBFile reader (e.g., [PyNWB](https://github.com/NeurodataWithoutBorders/pynwb) or [MatNWB](https://github.com/NeurodataWithoutBorders/matnwb))?

**Comment by bendichter** [comment]
Seems reasonable

**Comment by bendichter** [comment]
This makes more sense for time columns than it does for time stamps in a TimeSeries

**Comment by CodyCBakerPhD** [comment]
> This makes more sense for time columns than it does for time stamps in a TimeSeries

True, maybe it can be a flag in the util function, and each check can decide whether it is appropriate based on the type

**Comment by bendichter** [comment]
That sounds like a good approach

**Comment by pauladkisson** [comment]
Fixed by #480
